kani-driver/src/args/mod.rs

```diff
@@ -44,20 +73,20 @@ pub struct CargoKaniArgs {
     pub command: Option<CargoKaniSubcommand>,
 
     #[command(flatten)]
-    pub common_opts: KaniArgs,
+    pub verify_opts: VerifyArgs,
 }
 
 // cargo-kani takes optional subcommands to request specialized behavior
 #[derive(Debug, clap::Subcommand)]
 pub enum CargoKaniSubcommand {
     #[command(hide = true)]
-    Assess(crate::assess::AssessArgs),
+    Assess(Box<crate::assess::AssessArgs>),
 }
 
-// Common arguments for invoking Kani. This gets put into KaniContext, whereas
-// anything above is "local" to "main"'s control flow.
+// Common arguments for invoking Kani for verification purpose. This gets put into KaniContext,
+// whereas anything above is "local" to "main"'s control flow.
 #[derive(Debug, clap::Args)]
-pub struct KaniArgs {
+pub struct VerifyArgs {
```

Comment:
We have `ValidateArgs` the trait for validate arguments. Here `VerifyArgs` sounds like we are verifying the arguments. How about Rename it to `VerificationArgs`

Reply:
I really like that suggestion. Thanks!